content/about/feature-stages/index.md

```diff
@@ -74,7 +74,8 @@ Below is our list of existing features and their current phases. This informatio
 | [Kubernetes: Service Credential Distribution](/docs/concepts/security/#mutual-tls-authentication)   | Stable
```

Comment:
please split the feature-stages change into a separate change.  The TOC (iiuc) has to approve these.

Reply:
I removed feature stage change from this PR. Please take a look at it again. Thanks!

Reply:
I made a separate PR https://github.com/istio/istio.github.io/pull/1888 for feature stage change.